cmd/daemon/flux-notification-controller/api.go

```diff
@@ -0,0 +1,35 @@
+package flux_notification_controller
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"go.lunarway.com/zlog"
+)
+
+type Event struct {
+	InvolvedObject struct {
+		Kind            string `json:"kind"`
+		Namespace       string `json:"namespace"`
+		Name            string `json:"name"`
+		UID             string `json:"uid"`
+		APIVersion      string `json:"apiVersion"`
+		ResourceVersion string `json:"resourceVersion"`
+	} `json:"involvedObject"`
+	Severity            string    `json:"severity"`
+	Timestamp           time.Time `json:"timestamp"`
+	Message             string    `json:"message"`
+	Reason              string    `json:"reason"`
+	ReportingController string    `json:"reportingController"`
+	ReportingInstance   string    `json:"reportingInstance"`
+}
+
+func HandleEventFromFlux2(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+	body, _ := ioutil.ReadAll(r.Body) //TODO: log something
+	var event Event
+	_ = json.Unmarshal(body, &event)
+	zlog.Infof("Received alert from flux2-notification-controller: %s with msg: %s", event.InvolvedObject.Name, event.Message)
```

Comment:
Notice that `zlog` is not open source so we cannot use it here. Instead use the internal `github.com/lunarway/release-manager/internal/log` package.

```suggestion
	log.Infof("Received alert from flux2-notification-controller: %s with msg: %s", event.InvolvedObject.Name, event.Message)
```